pkg/client/replication-client_test.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func TestEnableVolumeReplication(t *testing.T) {
-	var client = NewReplicationClient(&grpc.ClientConn{}, time.Minute)
+	client := NewReplicationClient(&grpc.ClientConn{}, time.Minute)
```

Comment:
Please squash this commit and add this to the commit where you are fixing various linter issues

Reply:
Done.. @Madhu-1 ptal.. thanks